bin/elixir.bat

```diff
@@ -142,6 +143,7 @@ if ""==!par:--no-halt=!   (set "parsElixir=!parsElixir! --no-halt" && goto start
 if ""==!par:--remsh=!     (set "parsElixir=!parsElixir! --remsh %1" && shift && goto startloop)
 if ""==!par:--dot-iex=!   (set "parsElixir=!parsElixir! --dot-iex %1" && shift && goto startloop)
 rem ******* ERLANG PARAMETERS **********************
+if ""==!par:--app-env=!         (set "parsErlang=!parsErlang! -%1 %2 %3" && shift && shift && shift && goto startloop)
```

Comment:
For some bizarre reason, if I simply add more spaces to align this, the script breaks with `The system cannot find the batch label specified expand_erl_libs` :hear_no_evil: :thinking: